NuGet.config

```diff
@@ -18,5 +18,6 @@
   <packageSources>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="applicationinsights" value="https://www.myget.org/F/applicationinsights/api/v3/index.json" />
+    <add key="dotnet" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" protocolVersion="3" />
```

Comment:
Yes, .NET team agreed to release DiagSource when we will need it. I assume the deadline is around mid of April?

I created #487 to track it